README.md

```diff
@@ -137,7 +137,7 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   foreman start -f Procfile.dev
   ```
 
-7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world)
+7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world) or [localhost:5000/hello_world](http://localhost:5000/hello_world) if the former doesn't work.
```

Comment:
@justin808 we moved this to 3000 only. https://github.com/shakacode/react_on_rails/blob/master/lib/generators/react_on_rails/templates/base/base/Procfile.dev.tt

You're right.